webapp/src/components/issue_attribute_selector.jsx

```diff
@@ -32,7 +33,7 @@ export default class IssueAttributeSelector extends PureComponent {
     }
 
     componentDidMount() {
-        if (this.props.repoName) {
+        if (this.props.repoName && !this.props.disabled) {
```

Comment:
Does this change need to applied to `componentDidUpdate` too? I genuinely am not sure

Reply:
It looks like the components using this component (e.g. `GithubLabelSelector`) are also checking for the empty repo name and `disabled` prop in their `loadLabels` function.

One thing I want to make sure happens is that when we go from a repo that has valid options, to one that disables the component, we make sure to get the proper empty array of options applied to each respective component. This may require unconditionally calling the `loadLabels` function maybe?

It may not be an issue because we use `isDisabled` to disable the select box, but we also need to make sure we clear the available options and selected values too.

Reply:
Yes, not sure about the empty name. Used it as it was used.

Should not need anymore changes. Hi @dipak-demansol , could you please test the PR? and make sure it adheres to comment above. Thanks.

Reply:
@mickmister I have updated the code handle `handleRepoChange` for the comment regarding switching issues so that it resets the other fields.  I am asking @dipak-demansol  to QA it now but it seems to fix it.  
https://github.com/mattermost/mattermost-plugin-github/blob/917ace8e49f140be44187a3115ad0cf8c4e018d4/webapp/src/components/modals/create_issue/create_issue.jsx#L108-L119